### PR TITLE
Molecule radio button field component

### PR DESCRIPTION
**Pull request by juanmaguitar**
- ⚙︎Task → https://jira.scmspain.com/browse/SUIC-133

This PR full depends on this other PR → https://github.com/SUI-Components/sui-components/pull/799
So, no merge of this PR should be done until this other one (https://github.com/SUI-Components/sui-components/pull/799) is fully approved and merged

Also, no demo of this PR can be done until the component `AtomRadioButton` (created in https://github.com/SUI-Components/sui-components/pull/799) is deployed publicly as a NPM component

### CODE REVIEW

File: components/molecule/radioButtonField/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# MoleculeRadioButtonField
+
+> Description
```
Comment (jcger) [inline components/molecule/radioButtonField/README.md]: a little description and a real example would be nice, maybe @turolopezsanabria can help you with the first part

File: components/molecule/radioButtonField/src/index.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  label,
+  successText,
+  errorText,
+  helpText,
+  onChange,
+  ...props
+}) => {
+  const errorState = getErrorState(successText, errorText)
+  return (
+    <div className={BASE_CLASS}>
```
Comment (jcger) [inline components/molecule/radioButtonField/src/index.js:25]: is this div really needed, couldn't the moleculeField be the wrapper element?

File: components/molecule/radioButtonField/README.md (hunk 1 above)
Comment (juanmaguitar) [reply components/molecule/radioButtonField/README.md]: done! 👍

**Review by jordevo — APPROVED** [review]

File: components/molecule/radioButtonField/src/index.js (hunk 2 above)
Comment (juanmaguitar) [reply components/molecule/radioButtonField/src/index.js:25]: I think, that, in order to apply specific styles to elements inside this component, the right way to do it is to assign a specific class to this component, and we need a `div` to do that as we cannot assign a className directly to a `MoleculeField` component

**Review by jcger — APPROVED** [review]

**Review by kikoruiz — APPROVED** [review]

**Pull request merged**